strapi/strapi
# fix: hide the locale option in the Edit view dropdown if you don't have permission to read it

### What does it do?

To avoid weird behaviors in the Edit view, we have removed the option in the locale dropdown to filter the locale where the user has not read permission

### Why is it needed?

To solve a bug we had in v5, because the option was shown and if you click on it you have an error page

### How to test it?

 - Remove locale A permissions from Editor Role
 - Go to the Content Manager
 - Open an entry that has at least 2 locales
 - See that you have access to locale A as User A even without the permissions

### Related issue(s)/PR(s)
fixes https://github.com/strapi/strapi/issues/20838
